tests/configs/config_ERP_CORE.py

```diff
@@ -28,6 +28,7 @@
 
 study_name = 'ERP-CORE'
 bids_root = '~/mne_data/ERP_CORE'
+deriv_root = '~/mne_data/mne-bids-pipeline/tests/ERP_CORE'
```

Comment:
I would have even considered:
```suggestion
deriv_root = '~/mne_data/derivatives/mne-bids-pipeline/tests/ERP_CORE'
```
so we just move one step up the derivatives folder. But I won't fight for it.

Reply:
Yes I like this. Then we can drop the `test` subdir too:

```
'~/mne_data/derivatives/mne-bids-pipeline/ERP_CORE'
```